spec/rails_helper.rb

```diff
@@ -17,6 +17,7 @@
 # run twice. It is recommended that you do not name files matching this glob to
 # end with _spec.rb. You can configure this pattern with the --pattern
 # option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
+Dir[Rails.root.join('spec/support/*.rb')].each { |f| require f }
```

Comment:
most helpers were not being loaded since the glob pattern only matched files-within-subdirs of `spec/suppport`

This became necessary for me once I zapped the `rescue` lines in tests -- not sure why.